docs/topics/getting-started.md

```diff
@@ -166,7 +166,52 @@ Follow Kotlin on ![Twitter](twitter.svg){width=25}{type="joined"} [Twitter](http
 
 </tab>
 
+<tab title="Multiplatform library">
+
+Support for multiplatform programming is one of Kotlin’s key benefits. It reduces time spent writing and maintaining the same code for different platforms while retaining the flexibility and benefits of native programming.
+
+Here you'll learn how to develop and a multiplatform library:
+
+1. **Install the [latest version of IntelliJ IDEA](http://www.jetbrains.com/idea/download/index.html).**
+
+2. **Create a multiplaform library:**
+
+   * To start from scratch, [create a basic project](mpp-create-lib.md).
+   * If you prefer more robust examples, complete the [Create and publish a multiplatform library](multiplatform-library.md) tutorial. It shows how to create a multiplatform library for JVM, JS, and Native platforms, test it and publish to a local Maven repository.
+
+3. **Use libraries in your application.** Learn more about [adding dependencies](mpp-add-dependencies.md).
+
+   |Library|Details|
+   |-------|-------|
+   | Ktor |  [Docs](https://ktor.io/) and [sample](https://kotlinlang.org/docs/mobile/use-ktor-for-networking.html).| 
+   | Serialization |  [Docs](serialization.md) and [sample](https://play.kotlinlang.org/hands-on/Networking%20and%20Data%20Storage%20with%20Kotlin%20Multiplatfrom%20Mobile/04_Creating_a_data_model).|
+   | Coroutines |  [Docs](https://kotlinlang.org/docs/mobile/concurrency-overview.html) and [sample](https://kotlinlang.org/docs/mobile/concurrency-and-coroutines.html).|
```

Comment:
I see that the content from this table is copied from cross-platform tab of this page. However, I would like to pay your attention that the page by the link https://kotlinlang.org/docs/mobile/concurrency-overview.html contains more theoretical materials and mentions coroutines only twice as another way to perform some operations:
>freeze() is not available in shared Kotlin code, but several libraries provide expect and actual declarations for using it in shared code. However, if you're using a concurrency library, like kotlinx.coroutines, it will likely freeze data that crosses thread boundaries automatically.

>Concurrency libraries, like kotlinx.coroutines, will help automate freezing. You'll also be able to leverage the power of coroutines in your code and increase efficiency by sharing more code.


The second link which is called "sample" is much more suitable to be "Docs" – https://kotlinlang.org/docs/mobile/concurrency-and-coroutines.html. 

Maybe it's worth to throw the "sample" part of the description. At the moment I haven't found any coroutines samples.

Reply:
Fixed for coroutines.